src/hokusai/hokusai.yml

```diff
@@ -1,6 +1,6 @@
 # Orb Version 0.7.7
 
-version: 2.1
```

Comment:
I thought this was what [this message in the failed build](https://app.circleci.com/pipelines/github/artsy/orbs/431/workflows/1e70d075-d90a-4c0e-bf38-35b1fc8187b0/jobs/671) was asking me to do, but apparently not?

> artsy/hokusai has been updated since master but hasn't had its version bumped.
> Update its version in ./src/hokusai/hokusai.yml

Reply:
@pepopowitz - i think it's referring to the Orb [version](https://github.com/artsy/orbs/blob/60791e2a640eaa5fd9356cde3e1f33b625a9b450/src/hokusai/hokusai.yml#L1).

Reply:
a dev (canary) orb is [published](https://app.circleci.com/pipelines/github/artsy/orbs/432/workflows/27a04892-fbc0-46ba-92d5-931e1ee3305b/jobs/672). `artsy/hokusai@dev:af91d4c5df55b3a9712ac17c9ce3b549`. we can point convection to it.